### ISSUE TITLE
Level 1 Version 4 (v2.0.6): `logY` wrongly considered to be missing from a `curve` element

### ISSUE THREAD

**Issue by agarny** [fbergmann/libSEDML#63]
I am using [libSEDML v2.0.6](https://github.com/fbergmann/libSEDML/releases/tag/v2.0.6) and among other things, so far, I have gotten rid of the `logX` and `logY` attributes in a `curve` element since they are being deprecated in SED-ML L1V4. Instead, I am now using the new `xAxis`/`yAxis` and its `type` attribute to specify whether an axis uses a linear or a log10 scale ([here](https://github.com/fbergmann/libSEDML/files/3577094/noble_1962_remote.txt) is an example of one of my "new" SED-ML files).

However, if I know try to open that file, I get the following error messages:

```
/Users/Alan/Desktop/noble_1962_remote.txt
    [338:25] Error: A <curve> object must have the required attributes 'sedml:yDataReference' and 'sedml:type', and may have the optional attributes 'sedml:logY', 'sedml:xErrorUpper', 'sedml:xErrorLower', 'sedml:yErrorUpper' and 'sedml:yErrorLower'. No other attributes from the SBML Level 3 SEDML namespaces are permitted on a <curve> object. Reference: L3V1 Sedml V1 Section Sedml attribute 'logY' is missing from the <SedCurve> element.
    [427:1] Error: A <curve> object must have the required attributes 'sedml:yDataReference' and 'sedml:type', and may have the optional attributes 'sedml:logY', 'sedml:xErrorUpper', 'sedml:xErrorLower', 'sedml:yErrorUpper' and 'sedml:yErrorLower'. No other attributes from the SBML Level 3 SEDML namespaces are permitted on a <curve> object. Reference: L3V1 Sedml V1 Section Sedml attribute 'logY' is missing from the <SedCurve> element.
```

As can be seen:
 - `logY` is listed as being an optional attribute (why not deprecated? What about `logX`? In fact, we can see that `yDataReference` is rightly listed as a required attribute, but not `xDataReference`?).
 - a reference to "other attributes from the SBML Level 3 SEDML namespaces" is made. Why is SBML mentioned here?
 - a reference to "L3V1 Sedml V1 Section Sedml" is made, which doesn't mean anything in the context of SED-ML L1V4.
 - `logY` is considered to be missing the `SedCurve` element. For one, the message really ought to refere to the `curve` element rather than the name of the class. Secondly, `logY` is clearly not missing since it is deprecated in SED-ML L1V4.

Bottom line: I believe my "new" SED-ML file is valid L1V4 SED-ML, but I can read it using libSEDML 2.0.6.

**Comment by fbergmann** [comment]
agreed, that attribute should have been optional, and will now not flagged as missing. As far as the error messages are concerned, we can regenerate the messages, once l1v4 is finalized. At that point i'll make sure that no mentioning of SBML is in the final messages. As to why logX and xDataReference are not mentioned on the curve element, this is because they are read on the base class (AbstractCurve), and any error message would be coming from there.

**Comment by agarny** [comment]
Thanks a lot, @fbergmann, I can confirm that the issue with `logY` is now gone, so closing.
